cfme/cloud/provider/__init__.py

```diff
@@ -85,6 +88,27 @@ def is_displayed(self):
     including_entities = View.include(VMEntities, use_parent=True)
 
 
+class NetworkManagerDetailsView(NetworkProviderDetailsView):
```

Comment:
Why are these views defined here, and not in `cfme.networks.views`? They're network manager specific views.

Reply:
Not going to block merge, @ganeshhubale please consider a followup PR to move the classes.